### DISCUSSION TITLE
Home Assistant OS 14.0

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3711] [Announcements]
## Home Assistant Operating System

* Bump Buildroot to 2024.02.7 (#3659) @sairon
* Remove unnecessary kernel post-install hook for GRUB-based targets (#3672) @sairon
* Reduce timeout for network time synchronization to 15 seconds (#3669) @sairon
* Enable Cypress M8 USB serial driver for all targets (#3693) @sairon

## Raspberry Pi

* Add Hailo-8 firmware for Raspberry Pi AI Kit/HAT on RPi 5 (#3680) @sairon
* RaspberryPi: Update kernel to 6.6.51 - stable_20241008 (#3662) @sairon
* Disable UAS via usb-storage.quirks on RPi for JMicron JMS583 (#3687) @RubenKelevra

## Home Assistant Yellow

* Add initial support for Raspberry Pi CM5 on Yellow (#3667) @sairon
* RaspberryPi: Update kernel to 6.6.51 - stable_20241008 (#3662) @sairon
* Always use NVMe datadisk on Yellow if it's present on first boot (#3686) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Open Virtual Appliance

* Add iwlwifi-gl firmware needed for Intel BE200 Wi-Fi 7 card (#3668) @antst
* Add kernel config to support external i2c devices (#3642) @xtsrthm
* Add kernel options required for Atheros 12k wirelss adapters (#3640) @agners
* Linux: Update kernel to 6.6.62 (#3679) @sairon

## Generic x86-64

* Add iwlwifi-gl firmware needed for Intel BE200 Wi-Fi 7 card (#3668) @antst
* Add kernel options required for Atheros 12k wirelss adapters (#3640) @agners
* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Hardkernel ODROID

* Fall back to max eMMC clock to fix Kingston eMMC compat with ODROID-N2 (#3665) @sairon
* Linux: Update kernel to 6.6.63 (#3697) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Generic aarch64

* Add iwlwifi-gl firmware needed for Intel BE200 Wi-Fi 7 card (#3668) @antst
* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Build System

* Add Kconfig option for Supervisor channel (#3618) @agners
* Bump docker/build-push-action from 6.9.0 to 6.10.0 (#3704) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/14.0'>Home Assistant OS 14.0</a>.</em>

**Comment by pcmike** [comment]
Attempting to update an HAOS VM that previously when updating to 13.2 would get stuck on "target reached: system reboot" but would boot into 13.2 update if rebooted manually (and would subsequently reboot just fine from cli "host reboot") and now updating to 14 it's stuck this time at "Unmounting HassOS data partition..." :(

Threw in the towel and rolled back to my snapshot. I have no clue what's going on with these HAOS updates, but they're not applying cleanly since 13.2.

**Reply by RubenKelevra** [reply]
Hm technically the issue must have been introduced with 13.1 then, because that's what's shutting down, while 13.2 is written to the disk.

**Reply by agners** [reply]
@pcmike if you haven't can you open an issue and attach the output of `blkid` from the system console (use login to get OS shell access).

**Reply by agners** [reply]
In general, we obviously test every release, also on multiple different virtualization instances. Update worked for me fine from 13.1, to 13.2 and 14.0. So there must be something specific to your environment/setup.

**Reply by pcmike** [reply]
> @pcmike if you haven't can you open an issue and attach the output of `blkid` from the system console (use login to get OS shell access).

how can you actually get into the console? using novnc console from proxmox doesn't allow you to copy&paste and installing the ssh add-on and logging in via web ui or even regular ssh just drops you into some sort of environment that doesn't even have access to blkid. It just gives you access to "ha" and that doesn't get you into the actual system?

**Reply by pcmike** [reply]
> @pcmike if you haven't can you open an issue and attach the output of `blkid` from the system console (use login to get OS shell access).

https://github.com/home-assistant/operating-system/issues/3714#issue-2717920253

**Reply by Ladenburg1** [reply]
Same issue at my site.
Raspberry Pi 5 with nvme-storage
After pressing update the system starting a reboot but does not start. After unplugging from power the system starts with version 13.2

**Reply by RGarrett93** [reply]
I'm having the exact same issue, I'm stuck on 13.1: I've not been able to update beyond 13.2 and upto 14.0

My setup is Pi4 HAOS using USB SSD

**Reply by RubenKelevra** [reply]
> > @pcmike if you haven't can you open an issue and attach the output of `blkid` from the system console (use login to get OS shell access).
> 
> how can you actually get into the console? using novnc console from proxmox doesn't allow you to copy&paste and installing the ssh add-on and logging in via web ui or even regular ssh just drops you into some sort of environment that doesn't even have access to blkid. It just gives you access to "ha" and that doesn't get you into the actual system?

You need to disable the "Protection mode", to access the host via the add-on.

`blkid` will then still show no output, however you can list the ids via:

```
ls -l /dev/disk/by-uuid/ 
``` 

and for the list of mounts:

``` 
cat /proc/mounts
```

**Reply by RubenKelevra** [reply]
@Ladenburg1 / @RGarrett93 I'm running a Raspberry Pi 4B via USB-NVMe Storage - no issues here, so it's not a general issue, but something specific of your setup. I think it would be better to create new issues for the trouble you're having with all the details of your systems, like the type of SSD, the converter you're using etc.

**Reply by danwooller** [reply]
Mine hung but after a reboot came back at 14.0

**Reply by Ladenburg1** [reply]
Im using a NVMe-storage connected by a PCIe to NVMe SSD Shield (hat). It's an itenso 256 GB NVMe.
Trisd to update to 14 again with no success, hungs and after hard reboot it comes back with 13.2

**Reply by ctulhua** [reply]
Same here with RPi5 and nvme hat. Update 13.2 to 14 comes with success, but:
➜  ~ ha os info
board: rpi5-64
boot: A
boot_slots:
  A:
    state: booted
    status: good
    version: "13.2"
  B:
    state: inactive
    status: bad
    version: "14.0"
data_disk: TS1TMTE400S-I777670289
update_available: true
version: "13.2"
version_latest: "14.0"

**Comment by bearhntr** [comment]
Are there any plans to add the INTEL Bluetooth support?  For example the AX210/211 which is Wi-Fi 6E with Bluetooth.  More specifically for HAOS under Proxmox as a VM.

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

The system is running normally.

**Comment by thezfunk** [comment]
Anyone on a Pi 4 with a Z-wave USB stick want to volunteer to see if the boot loop issue is fixed?  Still on 11.5 with mine.

**Reply by Skuair** [reply]
I have a Pi3 with zigbee dongle-p and I had the boot loop issue since OS 12.4 only when host started with dongle plugged and zigbee2mqtt auto start. The workaround with the config.txt and dtoverlay=dwc2 worked for me.

**Comment by cristianstoica** [comment]
No issues with updating from 13.2 to 14.0 on RPi4.

**Reply by thezfunk** [reply]
Do you have a Z-wave USB stick?

**Comment by brugk000** [comment]
HP T630 no issues updating to HAOS14

**Comment by AndyFlysJabs** [comment]
I'm on 13.2, can see the update to 14.0 listed and have tried to upgrade 3 times. In all cases It hasn't occurred, but a full restart of the server it's on has occurred and I can't see anything in the system logs that infers an upgrade/update has failed.  At the Home Assistant Command line I can see its still 13.2

How can I find useful info that I can use to help you/me identify what the issue is?

I'm running the X86 image on an old Gen 8 HP Microserver.

Regards

Andy

**Reply by AndyFlysJabs** [reply]
Is the fact that I need to currently use GRUB2 to Allow and old HP Microserver to boot into 13.2 likely to be the cause of the 14 update failure....

**Reply by joro-home** [reply]
Same problem. Not able to upgrade HA OS 13.2 to 14. Also using GRUB2. Who can help me with a solution?

**Reply by Guruleenyc** [reply]
I just upgraded successfully on an x86 Lenovo m73

**Reply by joro-home** [reply]
> I just upgraded successfully on an x86 Lenovo m73

Also using Grub2? Or does it have a UEFI BIOS?

**Reply by joro-home** [reply]
As a new HA user I didn't know I had to select the boot-slot B manually. All works fine now.

**Comment by pwirkus** [comment]
Hello, my problem is as follows, I have an RPI5 8GB with an NVME disk and the update is successful, but after some time no ZigBee network works, all devices are not visible and I cannot enter the console as well as HACS and add-ons, then I have to turn off the power and wait until the system will load, I open the terminal and enter ha os update --version 13.2 and then everything works fine. I also cannot access the logs despite the message saying: the system says Error while loading pagr logs. Any advice on what could be wrong?

**Reply by akira215** [reply]
Hi,

Same issue here, seems to be a regression in HA OS 14.0, refer to [https://github.com/home-assistant/core/issues/132573](https://github.com/home-assistant/core/issues/132573l)
